services/brig/src/Brig/API/Public.hs

```diff
@@ -1193,6 +1196,31 @@ verifyDeleteUserH (r ::: _) = do
   API.verifyDeleteUser body !>> deleteUserError
   return (setStatus status200 empty)
 
+updateUserEmailValidation :: UserId -> UserId -> Handler ()
+updateUserEmailValidation zuserId emailOwnerId = do
+  maybeZuserTeamId <- lift $ Data.lookupUserTeam zuserId
```

Comment:
Maybe it is even cleaner, if we put the whole function into `MaybeT`?

Reply:
let's pair today!